### PR TITLE
Expanded use of Region class

### PR DESCRIPTION
**Pull request by jasonbaldridge**
This PR removes the region specific queries from Wikidata in favor of methods based on region corners that are defined centrally in cabby/geo/regions.py. This affected many files because the string names had to be changed to region lookups and the type signatures had to be fixed on a bunch of functions (from str to Region).

I also did some clean up in some files with respect to import order and unnecessary new lines here and there.

### CODE REVIEW

File: cabby/data/extract_test.py [hunk 1]
```diff
@@ -14,15 +14,13 @@
 
 '''Tests for wikidata.py'''
```
Comment (tzufgoogle) [inline cabby/data/extract_test.py:15]: Long before this PR. But just noticed it chould be chainged.

File: cabby/data/extract_test.py [hunk 2]
```diff
@@ -37,17 +35,26 @@ def testQueryItems(self):
     self.assertEqual(two_pnc_plazza.ref_instance, 'skyscraper')
   
   def testQueryWithOSM(self):
-    samples = extract.get_data_by_region_with_osm("Pittsburgh_small")
+    samples = extract.get_data_by_region_with_osm(
+        regions.get_region('Pittsburgh_small'))
+    for s in samples:
+      print(s)
+      print()
```
Comment (tzufgoogle) [inline cabby/data/extract_test.py:42]: Shouldn't this be removed?

File: cabby/data/extract_test.py (hunk 1 above)
Comment (tzufgoogle) [reply cabby/data/extract_test.py:15]: This test failed when I ran it.

**Review by tzufgoogle — COMMENTED** [review]
Looks great. 
There is only one test that seems to fail - extract_test.py

File: cabby/data/extract_test.py (hunk 2 above)
Comment (jasonbaldridge) [reply cabby/data/extract_test.py]: Yes, thanks for catching that!

File: cabby/data/extract_test.py (hunk 1 above)
Comment (jasonbaldridge) [reply cabby/data/extract_test.py]: Fixed the comment, good catch!

The test runs fine for me. Can you try doing the following?

```
$ bazel clean
$ ./build_all.sh
$ ./test_all.sh
```

**Review by jasonbaldridge — COMMENTED** [review]
Thanks for the review! The test does run fine for me, so please do bazel clean and then build and run all.

**Review by tzufgoogle — COMMENTED** [review]
LGTM

**Pull request merged**